### DISCUSSION TITLE
“found existing Vitess state on the external database”

### DISCUSSION THREAD

**Discussion by heejongahn** [planetscale/discussion#163] [Q&A] [answered]
- account email : heejongahn@gmail.com
- org name : galpi

Hi, I'm currently trying to import my AWS RDS into planetscale.

I made it work at the first time, but I accidentally activated 'primary mode'. As it was unintended action (I skimmed the texa and thought the button on second step would start Replica mode, not end it 😅) I cancelled the import (which triggered the removal of DB from planetscale console), and tried to re-connect to the same database.

However, now connection fails with `"Error checking server configuration: found existing Vitess state on the external database”` error message when I try to import the database again. What should I do to fix this?

Thanks in advance!

**Comment by Phanatic** [comment, marked as answer]
Hi  @heejongahn,

 Thank you for trying Data Imports into PlanetScale. The error message you're seeing is because some of the state from the previous import attempt was left behind in RDS. 

You can run this command with the mysql console connected to your RDS database : 

``` sql 
DROP DATABASE _vt;
```

After this, you can restart the import and it'll start completely anew.
